index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Link to the CSS file -->
+    <link rel="stylesheet" href="./index.css">
+    <title>Awesome Books</title>
+</head>
+
+<body>
+    <header>
+        <h2>Awesome Books</h2>
+        <ul id="navigation-menu">
+            <li><a id="list-link">List</a></li>
+            <span class="vertical-line"></span>
+            <li><a id="add-link">Add new</a></li>
+            <span class="vertical-line"></span>
+            <li><a id="contact-link">Contact</a></li>
+        </ul>
+    </header>
+    <h3 id="date-container"></h3>
+    
+    <div class="container">
+        <section class="book-list-container"><center><h1>All awesome Books</h1></center>
+            
+            <div class="book-list">
+                <ul id="book-list">
+    
+                </ul>
+            </div></section>
+        
+        
+        <section class="book-form">
+            <h2>Add a new book</h2>
+            <form id="book-form">
+                <input type="text" id="title" placeholder="Book Title">
+                <br>
+                <input type="text" id="author" placeholder="Book Author">
+                <br>
+                <div id="button-div"><button type="submit" id="submit-btn">Add</button></div>
+            </form>
+        
+        
+        
+        </section>
```

Comment:
- Hey, after adding the book, one doesn't seem to see the added books. please, let's fix and improve the app.

Reply:
@zieeco it is working 100% fine on My Laptop.

Reply:
It was Because of Luxon Dependency Library and Local Environment.